File: Additional Testing Code/NPMAPI/Models/StatmentDownloadRequestModel.cs
namespace NPMAPI.Models
{
    public class StatmentDownloadRequestModel
    {
        public long PracticeCode { get; set; }
        public long PatientAccount { get; set; }
        public string Message { get; set; }
        public long[] ExcludedClaimsIds { get; set; }
    }
}

File: Additional Testing Code/UserInterface/src/app/patient/Classes/statmentDownloadRequest.model.ts
export class StatmentDownloadRequest {
    PracticeCode: number;
    PatientAccount: number;
    Message:any;
    ExcludedClaimsIds: number[];
    constructor(private _PatientAccount?: number, private _ExcludedClaimsIds: number[] = []) {
        this.PatientAccount = _PatientAccount;
        this.ExcludedClaimsIds = _ExcludedClaimsIds;
    }
}